protocol/bin-prot/src/value/layout/traverse.rs

```diff
@@ -41,17 +41,8 @@ impl Iterator for BinProtRuleIterator {
                         // then call repeat the required number of times
                     }
                     BinProtRule::Record(mut rules) => {
-                        self.stack.extend(
-                            rules
-                                .drain(0..)
-                                .map(|field| {
-                                    if field.field_name == "current_protocol_version" {
```

Comment:
thanks for cleaning this up

Reply:
Caught it in benchmark tests as it prints tons of debug info during bench runs